examples/pytorch/graphsage/train_full.py

```diff
@@ -167,5 +83,42 @@ def main(args):
                         help="Aggregator type: mean/gcn/pool/lstm")
     args = parser.parse_args()
     print(args)
+    
+    # load and preprocess dataset
+    data = load_data(args)
```

Comment:
I prefer directly import the dataset classes from `dgl.data`. `load_data` is just an additional layer of abstraction and it is not applicable for datasets other than Cora, Citeseer, Pubmed. If we believe that they should be loaded with a single interface, then we should use a single class instead.

Reply:
Okay. See the above comment.  I will replace all load_data to dgl.data